src/actions/authenticate.tsx

```diff
@@ -0,0 +1,8 @@
+export const LOG_IN = "LOG_IN";
+
+export function userHasAuthenticated (val) {
+    return {
+      type: LOG_IN,
+      isLogin: val
```

Comment:
If you see `Login` as one word then everything else should be one word, ie `const LOGIN = "LOGIN"`